llvm/test/Transforms/OpenMP/replace_globalization.ll

```diff
@@ -201,7 +201,7 @@ declare void @unknown_no_openmp() "llvm.assume"="omp_no_openmp"
 ; CHECK-NEXT:    ret void
 ;
 ;
-; CHECK: Function Attrs: norecurse nosync nounwind allocsize(0) memory(read)
```

Comment:
If the PR is for better information in the presence of recursive call, why was this attribute dropped?

Reply:
> The "dropped" attributes are simply dropped because we do not query them anymore, not because we cannot derive them.

We never end up asking if this is norecurse, and OpenMP-opt doesn't derive this by default.